Localization/extended_kalman_filter/extended_kalman_filter.py

```diff
@@ -181,9 +190,9 @@ def main():
         time += DT
         u = calc_input()
 
-        xTrue, z, xDR, ud = observation(xTrue, xDR, u)
+        xTrue, xDR, z = observation(xTrue, xDR, u)
```

Comment:
Because previously `Qsim`/`Rsim` were wrongly applied to `u`, instead of `x`/`z`, resulting in `ud`.

This observation is also backed by the [example](https://en.wikipedia.org/wiki/Kalman_filter#Example_application,_technical.) given on Wikipedia as well as several other articles on KFs.

Reply:
hmm... I think it is impossible to use u in a real situation, because every sensor for input (like wheel encoder and IMU) has noise.